auto_sizing/size_calculation.py

```diff
@@ -107,8 +107,10 @@ def calculate_sample_sizes(
 
         result_dict = {
             key: {
-                "population_size": round(res[key]["population_size"], 2),
-                "population_percent": round(res[key]["population_percent"], 2),
+                "number_of_clients_targeted": round(res[key]["number_of_clients_targeted"], 2),
+                "population_percent_per_branch": round(
+                    res[key]["population_percent_per_branch"], 2
+                ),
```

Comment:
Maybe it doesn't matter, but if you think the extra decimal places would ever be useful, we can always handle the rounding on the UI side.